assets/scss/_variables_project.scss

```diff
@@ -24,201 +24,3 @@ $light-blue: #b6d0ff;
 
 // Fonts
 $google_font_family: "Open+Sans:300,300i,400,400i,600,600i,700,700i&display=swap" !default;
-
```

Comment:
This all should've been in `_styles_project.scss`, as to be able to make use of breakpoints and variables